wad_project/templates/socshare/society.html

```diff
@@ -2,3 +2,53 @@
 {% load static %}
 {% load template_tags %}
 
+{% block title %}
+Societies
+{% endblock %}
```

Comment:
Unnecessary block

Reply:
Those blocks correspond to the ../societies button